<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarkEditDialog</class>
 <widget class="QDialog" name="BookmarkEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,1,0,0">
   <item>
    <widget class="QLabel" name="lblName">
     <property name="styleSheet">
      <string notr="true">QLabel { font-weight: bold; }</string>
     </property>
     <property name="text">
      <string>Name:</string>
     </property>
     <property name="buddy">
      <cstring>leName</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item>
    <widget class="QLabel" name="lblURL">
     <property name="styleSheet">
      <string notr="true">QLabel { font-weight: bold; }</string>
     </property>
     <property name="text">
      <string>Location:</string>
     </property>
     <property name="buddy">
      <cstring>leURL</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="leURL">
     <property name="text">
      <string>http://</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblTags">
     <property name="styleSheet">
      <string notr="true">QLabel { font-weight: bold; }</string>
     </property>
     <property name="text">
      <string>Tags (space separated):</string>
     </property>
     <property name="buddy">
      <cstring>leTags</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TagLineEdit" name="leTags"/>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lblDesc">
          <property name="styleSheet">
           <string notr="true">QLabel { font-weight: bold; }</string>
          </property>
          <property name="text">
           <string>Description:</string>
          </property>
          <property name="buddy">
           <cstring>ptxDesc</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="ptxDesc"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="lblRating">
            <property name="styleSheet">
             <string notr="true">QLabel { font-weight: bold; }</string>
            </property>
            <property name="text">
             <string>Rating:</string>
            </property>
            <property name="buddy">
             <cstring>dialRating</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblRatingValue">
            <property name="text">
             <string>5.0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDial" name="dialRating">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="notchTarget">
           <double>12.500000000000000</double>
          </property>
          <property name="notchesVisible">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpFiles">
     <property name="styleSheet">
      <string notr="true">QGroupBox { font-weight: bold; }</string>
     </property>
     <property name="title">
      <string>Attached Files</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stwFileAttachments">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pageAttachedFiles">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableWidget" name="twAttachedFiles">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="btnShowAttachUI">
              <property name="text">
               <string>&amp;Attach File</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnSetFileAsDefault">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Set &amp;Default</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageAttachNew">
         <layout class="QGridLayout" name="gridLayout">
          <property name="verticalSpacing">
           <number>4</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblFileName">
            <property name="text">
             <string>FileName:</string>
            </property>
            <property name="buddy">
             <cstring>leFileName</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="leFileName"/>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="btnBrowse">
            <property name="text">
             <string>&amp;Browse...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="widget_2" native="true"/>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="chkRemoveOriginalFile">
            <property name="text">
             <string>Remove Original File After Attaching</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="btnAttach">
            <property name="text">
             <string>&amp;Attach</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QWidget" name="widget_3" native="true"/>
          </item>
          <item row="2" column="1">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>431</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="btnCancelAttach">
            <property name="text">
             <string>&amp;Cancel</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>chkRemoveOriginalFile</zorder>
         <zorder>btnAttach</zorder>
         <zorder>lblFileName</zorder>
         <zorder>leFileName</zorder>
         <zorder>btnBrowse</zorder>
         <zorder>horizontalSpacer_2</zorder>
         <zorder>btnCancelAttach</zorder>
         <zorder>widget_2</zorder>
         <zorder>widget_3</zorder>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagLineEdit</class>
   <extends>QLineEdit</extends>
   <header>TagLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>leName</tabstop>
  <tabstop>leURL</tabstop>
  <tabstop>leFileName</tabstop>
  <tabstop>btnBrowse</tabstop>
  <tabstop>btnAttach</tabstop>
  <tabstop>ptxDesc</tabstop>
  <tabstop>dialRating</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BookmarkEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BookmarkEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>390</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
